internal/service/credentials/resource_credential_issuance_rule_test.go

```diff
@@ -0,0 +1,661 @@
+
+  notification = {
+    methods = ["EMAIL", "SMS"]
+    template = {
+      locale  = "en"
+      variant = "template_B"
+    }
+  }
+}`, acctest.CredentialsSandboxEnvironment(), resourceName, name)
+}
+
+func testAccCredentialIssuanceRule_Minimal(resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+
+resource "pingone_credential_type" "%[2]s" {
+  environment_id       = data.pingone_environment.credentials_test.id
+  title                = "%[3]s"
+  description          = "%[3]s Example Description"
+  card_type            = "%[3]s"
+  card_design_template = "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 740 480\"><rect fill=\"none\" width=\"736\" height=\"476\" stroke=\"#CACED3\" stroke-width=\"3\" rx=\"10\" ry=\"10\" x=\"2\" y=\"2\"></rect><rect fill=\"$${cardColor}\" height=\"476\" rx=\"10\" ry=\"10\" width=\"736\" x=\"2\" y=\"2\" opacity=\"$${bgOpacityPercent}\"></rect><line y2=\"160\" x2=\"695\" y1=\"160\" x1=\"42.5\" stroke=\"$${textColor}\"></line><text fill=\"$${textColor}\" font-weight=\"450\" font-size=\"30\" x=\"160\" y=\"90\">$${cardTitle}</text><text fill=\"$${textColor}\" font-size=\"25\" font-weight=\"300\" x=\"160\" y=\"130\">$${cardSubtitle}</text></svg>"
+
+  metadata = {
+    name               = "%[3]s"
+    description        = "%[3]s Example Description"
+    version            = 5
+    bg_opacity_percent = 100
+    card_color         = "#000000"
+    text_color         = "#eff0f1"
+
+    fields = [
+      {
+        type       = "Alphanumeric Text"
+        title      = "Example Field"
+        value      = "Demo"
+        is_visible = false
+      },
+    ]
+  }
+}
+
+resource "pingone_application" "%[2]s" {
+  environment_id = data.pingone_environment.credentials_test.id
+  name           = "%[3]s"
+  enabled        = true
+
+  oidc_options {
+    type                        = "NATIVE_APP"
+    grant_types                 = ["CLIENT_CREDENTIALS"]
+    token_endpoint_authn_method = "CLIENT_SECRET_BASIC"
+    bundle_id                   = "com.pingidentity.ios_%[3]s"
+    package_name                = "com.pingidentity.android_%[3]s"
+
+    mobile_app {
+      bundle_id                = "com.pingidentity.ios_%[3]s"
+      package_name             = "com.pingidentity.android_%[3]s"
+      passcode_refresh_seconds = 30
+    }
+  }
+}
+
+resource "pingone_digital_wallet_application" "%[2]s" {
+  environment_id = data.pingone_environment.credentials_test.id
+  application_id = resource.pingone_application.%[2]s.id
+  name           = "%[3]s"
+  app_open_url   = "https://www.example.com"
+
+  depends_on = [resource.pingone_application.%[2]s]
+}
+
+resource "pingone_credential_issuance_rule" "%[2]s" {
+  environment_id                = data.pingone_environment.credentials_test.id
+  credential_type_id            = resource.pingone_credential_type.%[2]s.id
+  digital_wallet_application_id = resource.pingone_digital_wallet_application.%[2]s.id
+  status                        = "ACTIVE"
+
+  filter = {
```

Comment:
`notification` and `filter` are both optional in the schema, the minimal test should have neither set (ie, only the minimal required attributes)

Reply:
Changed the disabled test to cover this use case.  If issuance rule is active, then filter is required or the API call will fail (which exposed another validation I had to add to the resource).  The disabled test case covers notification & filter not defined (which has raised a new defect that I'm debugging).